packages/dataviews/src/stories/index.story.js

```diff
@@ -49,6 +49,20 @@ const fields = [
 		maxWidth: 400,
 		enableHiding: false,
```

Comment:
@oandregal I wanted to add a simple "text" field, Like an input where I can type the value or "start with" something ... I was surprised to find out that we don't support that yet and we only support "enumeration".

Reply:
Yes, all the current implementations use the global search instead. Added it to [the backlog and asked for design input](https://github.com/WordPress/gutenberg/issues/55100#issuecomment-1991824864).